cwms_radar_api/src/main/java/cwms/radar/api/SpecifiedLevelController.java

```diff
@@ -0,0 +1,128 @@
+package cwms.radar.api;
+
+import java.util.List;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+import javax.servlet.http.HttpServletResponse;
+
+import com.codahale.metrics.Histogram;
+import com.codahale.metrics.MetricRegistry;
+import com.codahale.metrics.Timer;
+import cwms.radar.api.errors.RadarError;
+import cwms.radar.data.dao.SpecifiedLevelDao;
+import cwms.radar.data.dto.SpecifiedLevel;
+import cwms.radar.formatters.ContentType;
+import cwms.radar.formatters.Formats;
+import io.javalin.apibuilder.CrudHandler;
+import io.javalin.core.util.Header;
+import io.javalin.http.Context;
+import io.javalin.plugin.openapi.annotations.OpenApi;
+import io.javalin.plugin.openapi.annotations.OpenApiContent;
+import io.javalin.plugin.openapi.annotations.OpenApiParam;
+import io.javalin.plugin.openapi.annotations.OpenApiResponse;
+import org.jetbrains.annotations.NotNull;
+import org.jooq.DSLContext;
+
+import static com.codahale.metrics.MetricRegistry.name;
+import static cwms.radar.data.dao.JooqDao.getDslContext;
+
+
+public class SpecifiedLevelController implements CrudHandler {
+    private static final Logger logger = Logger.getLogger(SpecifiedLevelController.class.getName());
+    private final MetricRegistry metrics;
+
+
+    private final Histogram requestResultSize;
+
+    public SpecifiedLevelController(MetricRegistry metrics){
+        this.metrics=metrics;
+        String className = this.getClass().getName();
+        requestResultSize = this.metrics.histogram((name(className,"results","size")));
+    }
+
+    @NotNull
+    protected SpecifiedLevelDao getDao(DSLContext dsl)
+    {
+        return new SpecifiedLevelDao(dsl);
+    }
+
+
+    private Timer.Context markAndTime(String subject)
+    {
+        return Controllers.markAndTime(metrics, getClass().getName(), subject);
+    }
+    
+    
+    @OpenApi(
+        queryParams = {
+            @OpenApiParam(name="office", required=false, description="Specifies the owning office of the Specified Levels whose data is to be included in the response. If this field is not specified, matching rating information from all offices shall be returned."),
+            @OpenApiParam(name="template-id-mask", required=false, description="Mask that specifies the IDs to be included in the response. If this field is not specified, all specified levels shall be returned."),
+        },
+            responses = {
```

Comment:
Indentation level doesn't match queryParams.
Nitpick, feel free to ignore. :-)

Reply:
np. I'll fix this.